README.md

```diff
@@ -42,6 +42,14 @@ If you have followed the instructions for [installing Kubeapps](./docs/user/gett
 - [Kubeapps components](./docs/architecture/overview.md)
 - [Roadmap](https://github.com/kubeapps/kubeapps/wiki/Roadmap)
 
+## API documentation
+
+Whereas Kubeapps does not have yet a well-design API, we still want to share the current status of the APIs used across the internal components. Check out the [latest API online documentation](https://app.swaggerhub.com/apis/kubeapps/Kubeapps) or download the Kubeapps [OpenAPI Specification yaml file](./dashboard/public/openapi.yaml) from the repository.
+
+> This API documentation is subject to change in each Kubeapps version. If you are planning to build any solution on the top of the Kubeapps API, please [reach us](https://kubernetes.slack.com/messages/kubeapps) for further information.
+
+Besides, you install Kubeapps in your cluster, you can find the interactive API documentation portal on the right menu.
```

Comment:
If you install Kubeapps in your cluster, ...

And "on the right menu" may not help people find it. Perhaps a screenshot?

Reply:
I thought so... but since it is the main readme, I didn't want to make any disruptive UI change there. Maybe we can remove it, no?

Reply:
Also, unrelated to this PR: should we remove the "Differences from Monocular" section since it is not that relevant? Or even move it to another doc, but not the main readme, no?

Reply:
> Also, unrelated to this PR: should we remove the "Differences from Monocular" section since it is not that relevant? Or even move it to another doc, but not the main readme, no?

yes, I'd remove it.